tests/e2e/vmgroups_test.go

```diff
@@ -66,7 +66,7 @@ func (vg *VmGroupTest) SetUpSuite(c *C) {
 		log.Printf(out)
 	}
 	// Create the test VM group1
-	cmd = adminconst.CreateVMgroup + vgTestVMgroup1 + " --default-datastore " + vg.config.Datastores[0]
+	cmd = adminconst.CreateVMgroup + vgTestVMgroup1 + " --default-datastore '" + vg.config.Datastores[0] + "'"
```

Comment:
So, could try out just changing GetTestConfig() to return the datastore names to prefix-suffix with _'_